spring-boot/074771ec125dd407af0282b92960e9e9e3377e84/original/README.md

```diff
@@ -26,28 +37,28 @@
     </plugin>
 ```
 
-## 5. Inside the folder  **spring-boot** run the command:
+## 7. Inside the folder spring-boot/spring-boot run the command:
     mvn clean compile -DskipTests assembly:single

```

Comment:
After applying the changes, I got an error when trying to generate the jar files for the **base** and **right** commits.

`Could not resolve dependencies for project org.springframework.boot:spring-boot:jar:1.2.5.BUILD-SNAPSHOT: Failure to find org.springframework.boot:spring-boot-configuration-processor:jar:1.2.5.BUILD-SNAPSHOT in https://repo.spring.io/snapshot was cached in the local repository, resolution will not be reattempted until the update interval of spring-snapshots has elapsed or updates are forced`

Reply:
@galilasmb @pauloborba

Reply:
@galilasmb , pensei que você tinha conseguido resolver isso

Reply:
@pauloborba refiz excluindo todas as dependências e funcionou. @leusonmario você executou dentro do caminho spring-boot/spring-boot para gerar o jar?

Reply:
@leusonmario I added one more step to check if version of spring-boot-parent is 1.2.5.BUILD-SNAPSHOT, change to version 1.2.5.RELEASE. Can you check if it works?

Reply:
I asked Matheus to run this steps on his machine, and tell me what happened.

Reply:
@galilasmb The new steps fixed the errors reported for the base and right commits.